src/proof/alethe/alethe_node_converter.cpp

```diff
@@ -34,11 +34,6 @@ Node AletheNodeConverter::postConvert(Node n)
                            << "\n";
       Node wi = SkolemManager::getOriginalForm(n);
       Trace("alethe-conv") << "AletheNodeConverter: ..original: " << wi << "\n";
-      if (wi == n)
```

Comment:
For now put an "Unreachable()" here saying that skolem support will be broken in Alethe. I'll do a PR once this one is merged in to fix it.

Reply:
There is already an unreachable on line 44. Or do you mean to also delete the block lines 37-43?

Reply:
Best to delete the whole block to handle skolems and put an unreachable. The fix for this is kinda lengthy so I'll submit in another PR (and there is no point in trying to make a somewhat workable version with the changes in this PR).